.gitignore

```diff
@@ -1,3 +1,4 @@
 .bitrise*
 _tmp
 .idea
+.env
```

Comment:
Why do you exclude the `.env` file?

Reply:
I use a .env file locally to define the env required for the Step execution, through this GoLand plugin: https://plugins.jetbrains.com/plugin/7861-envfile